doc/quickstart/re3gistry_quickstart.rst

```diff
@@ -0,0 +1,104 @@
+  * Migration: choose this option if you want to upgrade a pre-existing Re3gistry instance. This option will keep your settings and content. Note that this option will only work on an existing instance running on Re3gistry version 1.3.X.
+
+**Installation option**
+
+In case the "Installation" option is chosen, the basic configuration for the new registry will be requested.
+
+* System languages: the system languages are the languages supported and handled by the Re3gistry 2. Note that you can select one or more system languages.
+* Master language: the master language is the one used by the Re3gistry 2 to handle multilingualism. Note that this will require all content to be available in the chosen master language.
+* Base URI: the base URI will be part of the public URL, generally this should match with your own domain name: **http://example.com**/registry
+* Local ID: the local ID will be part of the public URL, generally the path following the domain name: http://example.com/**registry**
+* Label: the label of the registry will be the ‘title’ of the Re3gistry 2 instance. It will be shown in the registry’s landing page.
+* Content summary: this text should shortly describe what would be available under your Re3gistry 2 instance. It will be shown in the registry’s landing page.
+
+After filling all the information, click on "Save and continue".
+
+The next step will present a summary of the information inserted. If all the fields are correct, click on "Install" to begin the installation.
+Once the installation is finished, the user is redirected to the login page.
+
+**Migration option**
+
+In case the "Migration" option is chosen, the details of the database from the existing Re3gistry installation are requested.
+Fill the requested fields and click the "Save and continue" button. You will then get a page containing a summary of the data retrieved from the existing instance of the Re3gistry.
+Click on "Install" to begin the migration. An information page will be shown. Now you can close the browser and the system will notify you via e-mail using the address specified in the "Registry manager" definition step.
+
+Introduction to the Re3gistry 2
+================
+
+The Re3gistry editing interface allows to graphically manage the complete life-cycle of reference codes, such as code lists, enumerations, controlled vocabularies, taxonomies, thesauri or, simply, 'lists of things'.
+If you choose the "installation mode" from the wizard, the first time you log in to the system, you will have an empty registry. You can then start populating it using the tools provided by the editing interface.
+If you choose the "migration mode" from the wizard, you will find the registers imported from the older version of the tool.
+
+For detailed documentation on the editing interface, please read the `User manual: <https://github.com/ec-jrc/re3gistry/blob/master/documentation/user-manual.md>`
+
+Users, roles and groups
+==========
```

Comment:
Make 80 chars long preferably but at least make it at least as long as the title